controllers/openstackplaybookgenerator_controller.go

```diff
@@ -287,6 +287,11 @@ func (r *OpenStackPlaybookGeneratorReconciler) Reconcile(ctx context.Context, re
 		requeue, err := common.WaitOnJob(job, r.Client, r.Log)
 		r.Log.Info("Generating Playbooks...")
 		if err != nil {
+			// the job failed in error
+			deleteErr := r.Client.Delete(context.TODO(), heat)
+			if err != nil {
```

Comment:
Should this be `deleteErr` instead of `err`?

Reply:
Yes. Just pushed an update to fix this.